react/src/SearchVariants/components/SearchVariantsPage.tsx

```diff
@@ -21,11 +22,32 @@ const useStyles = makeStyles(theme => ({
 
 const SearchVariantsPage: React.FC<SearchVariantsPageProps> = () => {
     const [selectedGene, setSelectedGene] = useState<Gene>();
+    const [error, setError] = useState(false);
 
-    console.log(selectedGene);
-    // currently a dummy endpoint
-    /* eslint-disable @typescript-eslint/no-unused-vars */
-    //const { data: variants } = useVariantsQuery({ search: selectedGene }, "csv", !!selectedGene);
+    const { data: blob } = useVariantsQuery({ panel: selectedGene?.hgnc_gene_name }, "csv", {
+        enabled: !!selectedGene,
+        onError: e => {
+            //I'm not sure how common this scenario will be
+            if (e.status === 400) {
+                setError(true);
+            } else {
+                throw e;
```

Comment:
Maybe create a Snackbar error message here for potential 500 errors

Reply:
Can you give me steps to reproduce a 500? Thanks!

Reply:
Under `variant_summary` in `flask/app/gene_viewer.py`, add `abort(500, "Internal server error")` right below the function header.

In my case specifically I got a 500 because I tried importing Delvin's sqldump without migrating properly, so line 225 `sql_df = pd.read_sql(query.statement, query.session.bind)` failed because `analyses.priority` doesn't exist in my db. This was an issue specific to me, but showed that the frontend doesn't display 500 errors at all.

Reply:
updated in [b23a369](https://github.com/ccmbioinfo/stager/pull/518/commits/b23a369cb7de0f6db02749e40c81265bc1cee641)